cli/src/config.rs

```diff
@@ -652,6 +656,10 @@ cli_opt_struct! {
         #[clap(long)]
         solido_program_id: Pubkey,
 
+        /// Address of the Anker program.
+        #[clap(long, value_name = "address")]
+        anker_program_id: Pubkey,
```

Comment:
If we make it optional for `perform-maintenance`, we might as well make it optional here and keep things uniform.